src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -194,8 +190,10 @@ export const NavSearchBar: React.FC<NavSearchBarProps> = ({ venueId }) => {
     clearSearch,
   ]);
 
-  const numberOfSearchResults =
-    foundRooms.length + foundEvents.length + foundUsers.length;
+  const foundItems = useMemo<NavSearchListItem[]>(
+    () => [...foundRooms, ...foundEvents, ...foundUsers],
+    [foundRooms, foundEvents, foundUsers]
+  );
```

Comment:
[NB] The memos now produce only data object, not JSX elements. That's being moved to `<NavSearchList>`

Reply:
Let's revert this back to how it was previously.

Reply:
I'm also not sure that there's much/any real value of memoising this combined array here, based on how it's used in the latter component.